packages/react-components/src/echarts/extensions/yAxisSync/updateYAxis.ts

```diff
@@ -1,5 +1,6 @@
 import minBy from 'lodash.minby';
 import maxBy from 'lodash.maxby';
+import { isEqual } from 'lodash';
```

Comment:
Can we update this to use the same pattern as the above lodash methods? It enables treeshaking.